src/file-input/internal.tsx

```diff
@@ -20,6 +21,11 @@ import { FileInputProps } from './interfaces';
 
 import styles from './styles.css.js';
 
+interface InternalFileInputProps {
+  inputClassName?: string;
+  dataItemId?: string;
```

Comment:
I suggest we make it more generic and add `nativeInputAttributes?: React.InputHTMLAttributes<HTMLInputElement>` instead of dataItemId - same as we do in the Input component.

Reply:
`__nativeAttributes?: React.InputHTMLAttributes<HTMLInputElement> | Record<'data-${string}', string>;`

I updated it to be like this, which is how I saw it used in the InternalButton. Let me know if that looks alright.